statsmodels/tsa/stattools.py

```diff
@@ -931,9 +937,10 @@ def pacf(x, nlags=None, method="ywadjusted", alpha=None):
     Returns
     -------
     pacf : ndarray
-        Partial autocorrelations, nlags elements, including lag zero.
+        Partial autocorrelations, nlags+1 elements including lag zero.
     confint : ndarray, optional
-        Confidence intervals for the PACF. Returned if confint is not None.
+        Confidence intervals for the PACF, 2*(nlags+1) elements including lag
```

Comment:
Can you change here `Confidence intervals for the PACF. Has shape (nlags+1, 2) since it includes lag`

Reply:
Done, thanks